projects/addon-doc/src/components/navigation/navigation.template.html

```diff
@@ -48,6 +48,7 @@
                 *ngFor="let label of labels; index as index"
                 size="s"
                 [borders]="null"
+                [(open)]="!!openPagesArr[index]"
```

Comment:
Should it be just `[ ]` perhaps?

Reply:
not really, values in the `openPagesArr` should be synchronized with the UI state otherwise navigation to the particular item on searching will be misportrayed in case we closed the items group

Reply:
I don't think it works this way. You would need to write explicit openChange handler.

Reply:
@vadimk7 are you planning on returning to this in future? :) I'm going over all the PRs we have, looking at what can be merged and what has more work to do :)